llvm/test/CodeGen/AArch64/ptrauth-type-info-vptr-discr.ll

```diff
@@ -1,4 +1,4 @@
-; RUN: llc -mtriple aarch64-linux-gnu    -mattr=+pauth -filetype=asm -o - %s | FileCheck --check-prefix=ELF %s
+; RUN: llc -mtriple aarch64-linux-musl   -mattr=+pauth -filetype=asm -o - %s | FileCheck --check-prefix=ELF %s
```

Comment:
why is this needed? shouldn't the codegen for things that didn't need IRELATIVE before stay the same?

Reply:
That's needed because glibc doesn't support the PAuth relocations (so it uses IRELATIVE after this change), so I needed to switch to another triple that does.